index.html

```diff
@@ -425,7 +425,7 @@ <h2 id="steps">Steps for
 <span class="transwho" data-profile="CR|PR|PR-OBSL|PR-RSCND" data-cr="new|substantive|rec-update|rec-amended">
```

Comment:
```suggestion
<span class="transwho" data-profile="FPWD|CR|PR|PR-OBSL|PR-RSCND" data-cr="new|substantive|rec-update|rec-amended">
```

This is already the case, eg. https://lists.w3.org/Archives/Team/w3t-comm/2020Jan/0137.html